README.md

```diff
@@ -1,52 +1,234 @@
+
+2. Click  **Create credentials** and from the dropdown menu, select **OAuth client ID**.
+
+    ![Create Credentials](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-googleapis.calendar/tree/main/ballerina/resources/create-credentials.png)
+
+3. You will be directed to the **OAuth consent** screen, in which you need to fill in the necessary information below.
+
+    | Field                     | Value |
+    | ------------------------- | ----- |
+    | Application type          | Web Application |
+    | Name                      | CalendarConnector  |
+    | Authorized redirect URIs  | <https://developers.google.com/oauthplayground> |
+
+4. After filling in these details, click **Create**.
+
+5. Make sure to save the provided Client ID and Client secret.
+
+### Step 5: Get the Access and Refresh Tokens
+
+**Note**: It is recommended to use the OAuth 2.0 playground to obtain the tokens.
+
+1. Configure the OAuth playground with the OAuth client ID and client secret.
+
+    ![OAuth Playground](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-googleapis.calendar/tree/main/ballerina/resources/oauth-playground.png)
+
+2. Authorize the Calendar APIs.
+
+    ![Authorize APIs](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-googleapis.calendar/tree/main/ballerina/resources/authorize-calendar-apis.png)
+
+3. Exchange the authorization code for tokens.
+
+    ![Exchange Tokens](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-googleapis.calendar/tree/main/ballerina/resources/exchange-tokens.png)
+
+## Samples
+
+This sample demonstrates a scenario of creating a secondary calendar and adding a new event to it using the Ballerina Google Calendar connector.
+
+### Step 1: Import the package
+
+Import the `ballerinax/googleapis.calendar` package into your Ballerina project.
+
+```ballerina
+import ballerinax/googleapis.calendar;
+```
+
+### Step 2: Instantiate a new connector
+
+Create a `calendar:ConnectionConfig` with the obtained OAuth2.0 tokens and initialize the connector with it.
+
+```ballerina
+configurable string clientId= ?;
+configurable string clientSecret = ?;
+configurable string refreshToken= ?;
+configurable string refreshUrl = ?;
+
+calendar:Client calendarClient = check new ({
+   auth: {
+      clientId,
+      clientSecret,
+      refreshToken,
+      refreshUrl
+   }
+});
+```
+
+### Step 3: Invoke the connector operation
+
+You can now utilize the operations available within the connector.
+
+   ```ballerina
+   public function main() returns error? {
+      calendar:Client calendarClient = ...//
+
+      // create a calendar
+      calendar:Calendar calendar = check calendarClient->/calendars.post({
+         summary: "Work Schedule"
+      });
+
+      // quick add new event
+      string eventTitle = "Sample Event";
+      calendar:Event event = check calendarClient->/calendars/[calendarId]/events/quickAdd.post(eventTitle);
+   }
+   ```
```

Comment:
Check the indentations of the sample codes in MD files.

```suggestion
```ballerina
public function main() returns error? {
    calendar:Client calendarClient = ...//

    // create a calendar
    calendar:Calendar calendar = check calendarClient->/calendars.post({
        summary: "Work Schedule"
    });

    // quick add new event
    string eventTitle = "Sample Event";
    calendar:Event event = check calendarClient->/calendars/[calendarId]/events/quickAdd.post(eventTitle);
}
```
```